setup.py

```diff
@@ -30,23 +30,21 @@ def get_version():
 setup(
     name=PACKAGE,
     version=get_version(),
-    description="Quantum hardware backend for Qibo",
+    description="Quantum hardware module and drivers for Qibo",
     author="The Qibo team",
     author_email="",
     url="https://github.com/qiboteam/qibolab",
     packages=find_packages("src"),
     package_dir={"": "src"},
-    package_data={"": ["runcards/*.yml", "tests/*.yml", "*.txt"]},
-    entry_points={"console_scripts": ["calibrate-qubits = qibolab.calibration.calibrate_qubits:main"]},
+    package_data={"": ["runcards/*.yml", "tests/*.yml"]},
     zip_safe=False,
     classifiers=[
         "Programming Language :: Python :: 3",
         "Topic :: Scientific/Engineering :: Physics",
     ],
     install_requires=[
-        "qibo>=0.1.8rc0",
-        "qcodes",  # 0.34.1
-        "lmfit",  # 1.0.3
+        "qibo>=0.1.8",
+        "qcodes",
```

Comment:
Also `qcodes` could be moved into `tiiq` deps, right?

Reply:
I was going to ask the same, also I think qcodes is a dependency of qblox-instruments anyway, right?